charts/redpanda/values.yaml

```diff
@@ -92,9 +92,23 @@ external:
   # External config doesn't apply to RPC listeners as they are never externally accessible
   # These values can be overridden by each listener if needed
   enabled: true
-  # Default external access type (NodePort is the only option for now)
+  # Default external access type (only NodePort at the moment)
   type: NodePort
+  # Domain to be used for each node
   domain: local
+  #
+  # nodeType can be set to the following: subdomain ip
+  # with subdomain, each node will advertise an address with <external.nodes[replica index]>.<domain>
+  # with ip, each node will advertise only <external.nodes[replica index]> (and domain will be ignored)
+  nodeType: ip
```

Comment:
This changes the default behavior making it a breaking change.

Reply:
This was an accident to keep in (it is now commented out by default). Without this or `node` the current default behavior occurs.